lib/omniauth/strategies/openid_connect.rb

```diff
@@ -187,6 +204,22 @@ def public_key
         key_or_secret || config.jwks
       end
 
+      def pkce_authorize_params
+        options.pkce_verifier = SecureRandom.hex(64) if options.pkce_verifier.nil?
+
+        # NOTE: see https://tools.ietf.org/html/rfc7636#appendix-A
+        {
+          code_challenge: options.pkce_options[:code_challenge].call(options.pkce_verifier),
```

Comment:
Is `pkce_verifier` supposed to be passed in as a string or a `Proc`? This seems to imply it's really a string since the default `code_challenge` is just taking the digest of the string. Is the idea that you can specify a custom verifier that is called by the `code_challenge`?

Reply:
Yeah I think so, I didn't write this code so not sure. Seems overkill IMO as I can't think why you'd really want to specify a custom verifier impl.